frontend_app_workspace_advanced/src/component/SpaceSubscriptionsRequests.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import {
+  DropdownMenu,
+  formatAbsoluteDate,
+  NoHoverListItem,
+  UserInfo
+} from 'tracim_frontend_lib'
+
+export const SpaceSubscriptionsRequests = props => {
+  return (
+    props.subscriptionsRequests.length
+      ? (
+        <span className='workspace_advanced__subscriptionsRequest'>
+          {props.subscriptionsRequests.map(request =>
+            <NoHoverListItem
+              key={`${request.author.user_id}_${request.created_date}`}
+            >
+              <UserInfo
+                publicName={request.author.public_name}
+                username={request.author.username}
+              />
+
+              {request.state === 'pending' && (
+                <DropdownMenu
+                  buttonLabel={props.t('Manage request')}
+                  buttonCustomClass='outlineTextBtn primaryColorBorder primaryColorBgHover primaryColorBorderDarkenHover workspace_advanced__subscriptionsRequest__button'
+                  isButton
+                >
+                  <div
+                    onClick={() => props.onClickAcceptRequest(request.author.user_id)}
```

Comment:
Since you use it on both children, I suggest to add an onClick method to the DropdownMenu instead.
(and ideally use a button or a link in the DropdownMenu component if onClick is present)

Alternatively, you could use a link or a button here as the only child of DropdownMenu containing the two divs. Not sure using div inside buttons is valid HTML, so you may need to use a link or turn divs into spans.

That'll make the menu navigable with a keyboard.

Reply:
For now we have:

```
<DropdownMenu>
    <DropdownMenuItem>
        <child>
    </DropdownMenuItem>
    <DropdownMenuItem>
        <child>
    </DropdownMenuItem>
</DropdownMeu>
```

Where DropdownMenuItem is a button. The `onClick`method is different for each child, that's why I do not put at `DropdownMenu`...

Reply:
Sorry, I misunderstood the code. You can ignore this comment. That said, I think a link or a button would be more aproriate than a clickable div.